terraform/modules/gost_postgres/main.tf

```diff
@@ -29,10 +29,15 @@ resource "aws_rds_cluster_parameter_group" "postgres13" {
   }
 }
 
+resource "random_password" "postgres_user" {
+  length  = 32
+  special = false
+}
+
```

Comment:
This does force a new password to be generated, but should **not** invalidate existing connections, so it shouldn't cause any downtime.